doc/release/1.11.0-notes.rst

```diff
@@ -1,14 +1,27 @@
 NumPy 1.11.0 Release Notes
 **************************
 
-This release supports Python 2.6 - 2.7 and 3.2 - 3.5.
+This release supports Python 2.6 - 2.7 and 3.2 - 3.5 and contains a number
+of enhancements and improvements. Note also the build system changes listed
+below as they may have subtle effects.
+
+No Windows (TM) binaries are provided for this release due to a broken
+toolchain. One of the providers of Python packages for Windows (TM) is your
+best bet.
 
 
 Highlights
 ==========
 
-* The datetime64 type is now timezone naive. See "datetime64 changes" below
-  for more details.
+Details of these improvements can be found below.
+
+* The datetime64 type is now timezone naive.
+* A dtype parameter has been added to ``randint``.
+* Improved detection of two arrays possibly sharing memory.
+* Alternative algorithms for ``np.histogram`` bin estimation.
```

Comment:
These aren't alternative algorithms -- they are the first options to make it in numpy. I would say simply "automatic bin size estimation for np.histogram"